tauri-build/build.rs

```diff
@@ -0,0 +1,29 @@
+#[cfg(not(feature = "dev-server"))]
+extern crate tauri_includedir_codegen;
+
+#[cfg(not(feature = "dev-server"))]
+#[macro_use]
+extern crate serde_derive;
+#[cfg(not(feature = "dev-server"))]
+extern crate serde_json;
+
+#[cfg(not(feature = "dev-server"))]
+#[path = "src/config.rs"]
```

Comment:
Which part, the `serde` declarations?  Those are apparently needed because of scoping issues.  Config uses `serde` and if I remove the `serde_derive` and `serde_json` crates from build.rs it complains about not having access to those libraries.